syncopy/nwanalysis/connectivity_analysis.py

```diff
@@ -115,6 +115,10 @@ def connectivityanalysis(data, method="coh", keeptrials=False, output="abs",
     taper : str
         Only valid if `method` is `'coh'` or `'granger'`. Windowing function,
         one of :data:`~syncopy.specest.const_def.availableTapers`
+    taper_opt : dict or None
+        Dictionary with keys for additional taper parameters.
+        For example :func:`~scipy.signal.windows.kaiser` has
+        the additional parameter 'beta'.
```

Comment:
The `process_taper` routine is starting to get pretty complex with lots of if/else branching. If I understand correctly when utilizing DPSS tapers any user-provided `taper_opt` is ignored and instead `dpss_opt = {'NW': NW, 'Kmax': nTaper}` is used. I think this should be emphasized in the docstring.

Reply:
Yep, but at least it's re-usable for both `specest` and `nwanalysis` frontends ;) I am also not super happy about the current structure, but as you can see here; https://github.com/esi-neuroscience/syncopy/blob/f83188af815fe93ac84d2e18f4462cc326557c1f/syncopy/shared/input_processors.py#L246 The users are getting a warning, and `dpss` is only configurable via our `tapsmofrq` (and the not recommenden `nTaper`) 
I will amend the docstring!

Reply:
Great, thank you! Other than that, I think we can merge right away. Thanks @tensionhead !